components/PriceFilter.tsx

```diff
@@ -30,7 +30,7 @@ export default function PriceFilter({ handleApplyClick }: PriceFilterProps) {
       </div>
 
       <br />
-      <button onClick={() => handleApplyClick(min, max)}>적용</button>
+      <button onClick={() => handleApplyPriceFilterClick(min, max)}>적용</button>
```

Comment:
button에 type 속성을 잡아주는게 좋아요~

```suggestion
      <button onClick={() => handleApplyPriceFilterClick(min, max)} type='button'>적용</button>
```